defaults/test_default.yaml

```diff
@@ -30,6 +30,7 @@ reuse_cluster: false
 nemesis_class_name: 'NoOpMonkey'
 nemesis_during_prepare: true
 nemesis_interval: 5
+add_node_cnt: 3
```

Comment:
let's decrease to 1, since the add node and remove node( with decommission) are very time-consuming task that can trigger timeouts on some jobs and can prevent from other nemesis running (since they will consume the whole test run time)

Reply:
for specific tests with small datasets we can increase this to 3-5